### DISCUSSION TITLE
lack of module

### DISCUSSION THREAD

**Discussion by hulin-Lee** [TeamCOMPAS/COMPAS#932] [Q&A] [answered]
compasRootDir = os.environ['COMPAS_ROOT_DIR']
sys.path.append(compasRootDir + 'postProcessing/PythonScripts')
from compasUtils import printCompasDetails, getEventHistory, getEventStrings

When I try the above codes, I find that there is no module of "compasUtils" in the file I download. I wonder where I can find the module. Thanks a lot!

**Comment by SimonStevenson** [comment, marked as answer]
The compasUtils file currently resides in the 'untracked scripts' directory (see [here](https://github.com/TeamCOMPAS/COMPAS/tree/dev/utils/untracked_scripts)). Adding this directory to your path should resolve this error. We are working to make the Python infrastructure for COMPAS more user friendly (e.g., pip install-able).

**Reply by hulin-Lee** [reply]
Thanks again!
